packages/suite/src/storage/index.ts

```diff
@@ -15,15 +40,18 @@ const onUpgrade: OnUpgradeFunc<SuiteDBSchema> = async (db, oldVersion, newVersio
     if (oldVersion > 0 && oldVersion < 13) {
         // just delete whole db as migrations from version older than 13 (internal releases) are not implemented
         try {
-            await SuiteDB.removeStores(db);
+            SuiteDB.removeStores(db);
         } catch (err) {
             console.error('Storage: Error during removing all stores', err);
             throw err;
         }
     }
 
-    // migrate functions
-    await migrate(db, oldVersion, newVersion, transaction);
+    if (oldVersion < 57) {
```

Comment:
to make it really clear I would extract this to a local constant
```suggestion
    if (oldVersion < LAST_LEGACY_VERSION) {
```